apps/AEPSampleApp/ios/Podfile.lock

```diff
@@ -161,12 +161,10 @@ PODS:
   - RCTAEPEdgeIdentity (6.0.2):
     - AEPEdgeIdentity (< 6.0.0, >= 5.0.0)
     - React
-  - RCTAEPMessaging (6.0.2):
-    - AEPMessaging (< 6.0.0, >= 5.0.0)
-    - glog
-    - RCT-Folly (= 2022.05.16.00)
-    - React-Core
-  - RCTAEPOptimize (6.0.2):
+  - RCTAEPMessaging (6.0.3):
```

Comment:
Do you want to build the sample app with 6.0.4 version?

Reply:
I can upgrade it if youd like, that one came out after I had worked on this and is fresh off the press

Reply:
It is fine to leave it as is.